### PR TITLE
[453440][formatter2] hook up serializer

### PR DESCRIPTION
**Pull request by meysholdt**
https://bugs.eclipse.org/bugs/show_bug.cgi?id=453440

Signed-off-by: Moritz Eysholdt <moritz.eysholdt@itemis.de>

### CODE REVIEW

File: plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/AbstractHiddenRegion.java [hunk 1]
```diff
@@ -23,9 +23,11 @@
 /**
  * @author Moritz Eysholdt - Initial contribution and API
  */
+/**
+ * @author Moritz Eysholdt - Initial contribution and API
```
Comment (szarnekow) [inline plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/AbstractHiddenRegion.java]: Duplicate comment?

**Comment by oehme** [comment]
Jenkins xtend-runtime-tests-master-latest

**Comment by oehme** [comment]
Jenkins xtend-runtime-tests-master-latest

**Comment by oehme** [comment]
Jenkins xtend-runtime-tests-master-latest

File: plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/AbstractHiddenRegion.java (hunk 1 above)
Comment (meysholdt) [inline plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/internal/AbstractHiddenRegion.java]: fixed in 5d7d4ef

**Comment by szarnekow** [comment]
Nice to see the outcome of that change directly, e.g. when applying quickfixes :+1:

**Comment by meysholdt** [comment]
yes indeed :-)

**Pull request merged**